src/pipeline_edsl/to_workflow.ml

```diff
@@ -284,6 +284,21 @@ module Make (Config : Compiler_configuration)
           ~name:(sprintf "Input file: %s" url)
       in
       Raw_file (deal_with_input_file raw_file (single_file ~host))
+    | Some "http" | Some "https" ->
+      let path =
+        let basename =
+          match Uri.get_query_param uri "filename" with
```

Comment:
This is just some bonus functionality, right? I.e. I want to make sure my file resides at a particular path once downloaded?